build.sh

```diff
@@ -204,18 +228,25 @@ cmake_boolean_option()
 	esac
 }
 
+# ---------------------------------------------------------------------------
+# Let's process the user's wishes
+# ---------------------------------------------------------------------------
+
+features_set_to_autodetect
+parse_args "$@"
+
+if [ $PRINT_HELP -ne 0 ] ; then
+	print_help
+	exit 1
+fi
+
+MAKE_TASKS=$(num_cpu)
```

Comment:
Oops i didn't see that comment in my email.

It was easier to just move the MAKE_vars lines above features_set_to_autodetect call.

Too bad you merged fast. I'll just modify directly the script as it's a very small change, it makes the code more straightforward